client/hooks/useMuncher.js

```diff
@@ -3,21 +3,21 @@ import newGame from '../GameState/muncherState.js';
 import useWords from './useWords.js';
 
 export function useMuncher() {
+  const [gameover, setGameover] = useState(false);
```

Comment:
Any thought towards using context for some of this, seems like a decent use case?